acceptance/tests/lookup/config5_interpolation.rb

```diff
@@ -2,13 +2,16 @@
   require 'puppet/acceptance/environment_utils.rb'
   extend Puppet::Acceptance::EnvironmentUtils
 
+
 tag 'audit:high',
     'audit:integration',
     'audit:refactor',  # This test specifically tests interpolation on the master.
                        # Recommend adding an additonal test that validates
                        # lookup in a masterless setup.
     'server'
 
+  pending_test('unexpected `::roles` returning undefined here, but the test passes when strict=warning')
```

Comment:
could you add `PUP-11751` to the message, so I remember to revisit this test

Reply:
I updated the comment to specifically look at the acceptance test too.